.ci/verify

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cd "$(dirname $0)/.."
+
+git config --global user.email "gardener@sap.com"
+git config --global user.name "Gardener CI/CD"
+
+mkdir -p /go/src/github.com/gardener/gardener-extension-runtime-gvisor
```

Comment:
This is only needed if the `k8s.io/code-generator` is used. I don't think this is the case for this project. Remove it.

Reply:
You mean lines 9 -14 ? So just execute make?

Reply:
Forget it, I only saw later that you use the code generator. Thus, you have to keep it. Though, you could add the same TODO like in other extensions, ref https://github.com/gardener/gardener-extension-provider-aws/blob/master/.ci/verify#L12